ros2bag/ros2bag/verb/play.py

```diff
@@ -162,6 +162,9 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             help='Determine the source of the service requests to be replayed. This option only '
                  'makes sense if the "--publish-service-requests" option is set. By default,'
                  ' the service requests replaying from recorded service introspection message.')
+        parser.add_argument(
+            '--log-level', type=str, default='info', choices=['debug', 'info', 'warn', 'error', 'fatal'],
+            help='Logging level.')
```

Comment:
How about moving this into the method like `add_loglevel_argument(parser)` to `api/__init__.py`?

```python
def add_loglevel_argument(parser):
    parser.add_argument(
       '--log-level', type=str, default='info',
       choices=['debug', 'info', 'warn', 'error', 'fatal'],
       help='Logging level'
    )
```

and we can call it here,

```suggestion
        add_loglevel_argument(parser)
```

so that this can be imported and used on else where, possibly on `Recorder` as well?

Reply:
@fujitatomoya I would prefer to have a parameter's definition in one place for the sake of transparency. 
If one needs to look at or check what parameters are defined or exist, if it will be a call to the `add_loglevel_argument(parser)` will need to have an extra headache and "archeology search" to figure out where it is defined to see what is inside of it.
Duplication of the same parameter definitions for player and recorder is worth it here for clarity.